src/chapters/functions/studio.rst

```diff
@@ -0,0 +1,169 @@
+Studio: Shapes
+==============
+
+To become good at solving problems with code you need to be able to break large problems into small ones. Usually, these smaller problems will take the form of functions that are used to solve the larger problem. Therefore, to become good at solving problems you need to become good at writing functions. And to master functions, you need to write a *lot* of them.
+
+This studio will have you write lots of relatively small functions, which combine to form larger, more complicated functions.
+
+At the end, you will be able to create strings of shapes, like this nifty diamond:
+
+::
+
+       #     
+      ###    
+     #####   
+    #######  
+   #########   
+    #######    
+     #####     
+      ###      
+       #
+
+There is no starter code for this studio, so create a new Node.js project at `repl.it <https://repl.it/student>`_ to get started.
+
+Rectangles
+----------
+
+#. Write a function ``line(size)`` that returns a line with exactly ``size``
```

Comment:
So Python has ``print('#'*size)`` to get a line of #s (``print('#'*5)`` = '#####').  Is there a console.log way to do this, or do the students need to use the ``repeat`` string method?  Is ``repeat`` on the string methods page?

Reply:
In my solution I use a `for` loop with the accumulator pattern, since that's what I imagined most students would do. I wouldn't expect them to know `String.repeat`, and no, there isn't a version of the `*` operator for this in JS like there is in Python.